vignettes/wip/dm-db.Rmd

```diff
@@ -54,7 +62,17 @@ my_dm <- dm_from_src(my_db)
 my_dm
 ``````
 
-- Flexible creation from individual tables or views
+The components of the my_dm object are lazy tables powered by {[dbplyr](https://dbplyr.tidyverse.org/)}.
+{dbplyr} translates the {[dplyr](https://dplyr.tidyverse.org/)} grammar of data manipulation into queries the database server understands.
+Lazy tables defer downloading of table data until results are collected for printing or local processing.
+
+## Building a dm from tables
+
+A dm can also be constructed from individual tables or views.
+This is useful for when you want to work with a subset of a database's tables or you want to build a database out of tables from different sources.
```

Comment:
We don't currently support combining tables from multiple sources in a single dm.

```suggestion
This is useful for when you want to work with a subset of a database's tables, perhaps from different schemas.
```